addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -551,5 +551,69 @@ odoo.define('website.tour.form_editor', function (require) {
         }
     ]);
 
+    function editContactUs(steps) {
+        return [
+            {
+                content: "Enter edit mode",
+                trigger: 'a[data-action=edit]',
+            }, {
+                content: "Select the contact us form by clicking on an input field",
+                trigger: '.s_website_form input',
+                extra_trigger: '#oe_snippets .oe_snippet_thumbnail',
+                run: 'click',
+            },
+            ...steps,
+            {
+                content: 'Save the page',
+                trigger: 'button[data-action=save]',
+            },
+            {
+                content: 'Wait for reload',
+                trigger: 'body:not(.editor_enable)',
+            },
+        ];
+    }
+
+    tour.register('website_form_contactus_edition_with_email', {
+        test: true,
+    }, editContactUs([
+        {
+            content: 'Change the Recipient Email',
+            trigger: '[data-field-name="email_to"] input',
+            run: 'text test@test.test',
+        },
+    ]));
+    tour.register('website_form_contactus_edition_no_email', {
+        test: true,
+    }, editContactUs([
+        {
+            content: "Change a random option",
+            trigger: '[data-set-mark] input',
+            run: 'text_blur **',
+        },
+    ]));
+    tour.register('website_form_contactus_submit', {
+        test: true,
```

Comment:
Missing URL? I don't think there is a reason to omit it, it can't be run manually without it, even if the test(s) is(are) straigthforward and quite small, I'd still add it

Reply:
Sure, I copy-pasted the other tests of this file which do not have URL either :grimacing: